tests/copp/test_copp.py

```diff
@@ -191,12 +191,17 @@ def _gather_test_params(tbinfo, duthost, request):
                                topo=topo,
                                bgp_graph=bgp_graph)
 
-def _setup_testbed(dut, creds, ptf, test_params):
+@pytest.mark.usefixtures('disable_container_autorestart')
+def _setup_testbed(dut, creds, ptf, test_params, \
+        disable_container_autorestart):
     """
         Sets up the testbed to run the COPP tests.
     """
 
     logging.info("Disable LLDP for COPP tests")
+    feature_list = ['lldp']
+    disable_container_autorestart(testcase="test_copp", feature_list=feature_list)
```

Comment:
What is the reason to disable only lldp auto-restart instead of all?

Reply:
This is due to the lines below where we execute `docker stop` only for `lldp`. So that after stopping `lldp`, it doesn't restart.